docs/schedule/reference.es.md

```diff
@@ -117,6 +118,7 @@ Esta especificación define los siguientes archivos:
 | [calendar_dates.txt](#calendar_datestxt)           | **Condicionalmente obligatorio** | Las excepciones para los servicios definidos en el [calendar.txt](#calendartxt). <br /><br />Condicionalmente Obligatorio:<br /> - **Exigida** si [calendar.txt](#calendartxt) se omite. En cuyo caso [calendar_dates.txt](#calendar_datestxt) debe contener todas las fechas de servicio. <br /> - Opcional en caso contrario.                                                                                                                                                                |
 | [fare_attributes.txt](#fare_attributestxt)         | Opcional                         | Información sobre las tarifas de los itinerarios de una agencia de transporte.                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | [fare_rules.txt](#fare_rulestxt)                   | Opcional    | Normas de aplicación de las tarifas para los itinerarios.                                                                                                                                                                                                                                                                           |
+| [fare_media.txt](#fare_mediatxt) | Opcional | Describir los medios tarifarios que pueden emplearse para utilizar productos tarifarios.<br/><br/>El archivo [fare_media](#fare_mediatxt).txt describe conceptos que no están representados en [fare_attributes.txt](#fare_attributestxt) xt y fare_rules [fare_rules.txt](#fare_rulestxt)txt. Como tal, el uso de [fare_media](#fare_mediatxt).txt es totalmente independiente de los archivos [fare_attributes.txt](#fare_attributestxt) xt y fare_rules. [fare_rules.txt](#fare_rulestxt). |
```

Comment:
First sentence:
- .txt not included in hyperlink in first  fare_media.txt
- extra xt after fare_attributes.txt hyperlink
- extra fare_rules before and extra txt after fare_rules.txt

Second sentence:
- extra xt after fare_attributes.txt
- extra fare_rules before and extra txt after fare_rules.txt